src/finch/tensor.py

```diff
@@ -683,34 +686,53 @@ def full_like(
     *,
     dtype: DType | None = None,
     format: str = "coo",
+    device: Literal["cpu"] | None = None,
```

Comment:
```suggestion
    device: Device = None,
```

And let's define `Device` in `finch.typing`?

Reply:
Done!